src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -215,12 +215,6 @@ public static async System.Threading.Tasks.Task ProcessLogFileAsync(string fileP
             }
 
             await ProcessSarifLogAsync(log, outputPath, showMessageOnNoResults: promptOnLogConversions, cleanErrors: cleanErrors).ConfigureAwait(continueOnCapturedContext: false);
-
-            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
-            if (AsyncPackage.GetGlobalService(typeof(DTE)) is DTE2 dte)
-            {
-                dte.ExecuteCommand("View.ErrorList");
```

Comment:
>ErrorList [](start = 41, length = 9)

So if we don't do this, when does the viewer see the error list? I understand you don't want to pull focus to it, but if (1) if it wasn't previously opened you do want to open it, and (2) if it's open but not the currently selected tab in -- in whatever you call that collection of windows at the bottom -- then you do want to make that the selected tab. IMO.

Reply:
I see your PR description explicitly says "It's ok to "show" the Error list"... so I will keep looking for the code that does that.

---
In reply to: [504119046](https://github.com/microsoft/sarif-visualstudio-extension/pull/223#discussion_r504119046) [](ancestors = 504119046)